src/App.js

```diff
@@ -72,6 +73,34 @@ function App() {
     setNotes(notes)
     onCloseModal()
   }
+  const [openDeleteModal, setOpenDeleteModal] = useState({
+    show: false,
+    id: null,
+    titleNote: ""
+  });
+  const handleDialog = (show, id, titleNote) => {
+    setOpenDeleteModal({
+      show,
+      id,
+      titleNote
+    })
+  }
+  const deleteNote = (id) => {
+   const index = notes.findIndex((note) => note.id === id)
+   handleDialog(true, id, notes[index].title)
+  }
+  const deleteNoteTrue = (id, titleNote) => {
```

Comment:
`deleteNote` instead of `deleteNoteTrue`
and receive only id. Id is enough to delete the note